src/components/_list-view.scss

```
@@ -0,0 +1,321 @@
+/*doc
+---
+title: ListView
+name: list-view
+category: Components
+---
+
+選択可能なリストを表現するコンポーネントです。
+1ラインから3ラインまでのリストを作成することが可能で、`.ncgr-list-view`に`-two-line`、`three-line`のmodifierを付与することでそれに沿った形になります。
+また、リストの上下paddingを1remから0.5remに変更する`-dense`modifierが存在します。
+
+リスト内の要素は、ユーザーアイコンなどの要素を配置できる`leading`と、テキスト用の`title`、アクションボタン等ほかの要素を配置できる`trailing`に分かれています。
+
+```example
+<section>
```

[IMO]ここも特に当てるスタイルはないんだけど、一つのまとまりということをわかりやすくするために `.ncgr-list-group`を当ててもいいかも

参考：https://material.io/develop/web/components/lists#list-groups

```suggestion
<section class="ncgr-list-group">
```